3_Automation/06_Bestiary.yml

```
@@ -0,0 +1,598 @@
+Name:                                 # must be unique
+  Type: Boss, Dealer, Bystander, etc. # Required field
+  HP: X                               # Required field
+  AR: X                               # Required field
+  FP: X                               # Required field
+  Skill Mods:                         # List only those that apply
+    AGL: ±X                           # + is implied if not explicit
+    CON: ±X
+    GUT: ±X
+    INT: ±X
+    STR: ±X
+    VIT: ±X
+  Passives:
+    Passive: Descrip                  # Any text here
+  Vulnys:
+    Vulny: Descrip
+  Actions:
+    Name:                             # Note: for Companions, type is action for Dealer
+      Type: Major, Minor, Adversary, Free, House # Required.
+      Trigger: Descrip                # Condition of Free Actions
+      To hit: ±X
+      Damage: X
+      Damage type:                    # e.g., if poison, implies bypass AP
+      Range: X
+      AOE: (aura, cone, line) X       # line would be X by Y, requires new diagram
+      Target: X                       # number: 1 for single, 2 for 2 targets
+      Effect: Descrip
+      Save:
+        Trigger: Once, On hit, Start turn, etc
+        DR: X
+        Type: MOD                     # AGL, CON, etc
+        Fail: Descrip                 # effect on fail
+        Succeed: Descrip              # effect on success. Assume no access to above
+  Threat Level: X                     # Consistent comparable rating fitting formula
+  Phases:                             # only for Boss
+    One:
+      HP: X
+      Allies:
+  Element: Earth, Fire, etc           # Setting-specific
+  Description: Descrip                # Full descriptive text
+
+
+  Falcon:
+  Type: Companion, level 1
+  HP: 1
+  Passives:
+    Keen Eyed: When their Dealer makes a Detection check as an action with the companion present, they do so with Upper Hand
+  Action:
+    Swoop:
+      Type: Minor
+      Target: 1
+      Effect: Next attack against the target has Upper Hand
+  Element: Air
+
+Bear:
+  Type: Companion, level 1
+  HP: 3
+  Actions:
+    Bear Hug: 
+      Type: Minor
+      Range: 0
+      Target: 1
+      Effect: Grapples target enemy
+      Save:
+        Trigger: Once
+        DR: 2
+        Type: STR 
+        Fail: All attacks against the target made until the end of their next turn have Upper Hand.
+  Element: Earth
+
+Wolf:
+  Type: Companion, level 1
+  HP: 2
+  Passive:
+    Mount: When ridden, grants +3 to rider.
+  Actions:
+    Pack attack:
+      Type: Minor
+      Range: 0
+      Effect: If they are next to at least 2 enemies, all attacks against the target until the end of their turn have Upper Hand.
+  Element: Fire
+
+Horse:
+  Type: Companion, level 1
+  HP: 2
+  Passive:
```

the top-level item is plural in the template I think. If the spelling differs across instances, they'll register as different things to any code reading it
```suggestion
  Passives:
```